### PR TITLE
BLO-604 common container UI

### PR DESCRIPTION
**Pull request by simonheys**
In preparation for future animations, this PR updates the layout (not design) of remaining screens to better inherit their dimensions from their parent container. This is achieved either using the common navigation container component, or where the screens are 'novel', removing local explicit viewport sizes and replacing with flex equivalents. Some position:fixed elements remain, these should be removed in a future PR once their behaviour in an animated environment is better understood.

### CODE REVIEW

File: packages/extension/src/ui/features/accountNfts/SendNftScreen.tsx [hunk 1]
```diff
@@ -68,16 +67,23 @@ export const NftImageContainer = styled.div`
 `
 export const StyledForm = styled.form`
```
Comment (bluecco) [inline packages/extension/src/ui/features/accountNfts/SendNftScreen.tsx:68]: waiting for future design to remove `styled`?

File: packages/extension/src/ui/features/accountNfts/SendNftScreen.tsx [hunk 2]
```diff
@@ -68,16 +67,23 @@ export const NftImageContainer = styled.div`
...
-  height: 87vh;
   display: flex;
+  flex: 1;
   flex-direction: column;
   justify-content: space-between;
   gap: 12px;
   width: 100%;
 `
+
+const ButtonSpacer = styled.div`
```
Comment (bluecco) [inline packages/extension/src/ui/features/accountNfts/SendNftScreen.tsx:78]: same

File: packages/extension/src/ui/features/actions/AddNetworkScreen.tsx [hunk 3]
```diff
@@ -14,6 +14,7 @@ import { recover } from "../recovery/recovery.service"
 
 const AddTokenScreenWrapper = styled.div`
```
Comment (bluecco) [inline packages/extension/src/ui/features/actions/AddNetworkScreen.tsx:15]: remove `styled` or waiting for new design?

File: packages/extension/src/ui/features/actions/AddTokenScreen.tsx [hunk 4]
```diff
@@ -23,12 +22,14 @@ import { useTokensInNetwork } from "../accountTokens/tokens.state"
 
 const AddTokenScreenWrapper = styled.div`
```
Comment (bluecco) [inline packages/extension/src/ui/features/actions/AddTokenScreen.tsx:23]: remove `styled` or waiting for new design?

Comment (simonheys) [reply packages/extension/src/ui/features/actions/AddTokenScreen.tsx:23]: Exactly, this PR is just updating the containers.

**Review by bluecco — APPROVED** [review]

**Pull request merged**